setup.py

```diff
@@ -19,7 +19,7 @@
       packages=find_packages(exclude=['ez_setup']),
       include_package_data=True,
       install_requires=[
-          'pymongo >= 2.3, < 3',
+          'datastore',
```

Comment:
I'd add a >= constraint here, just to document the minimum supported version of datastore.